### PR TITLE
Fix how to get the space ID documentation

### PR DESCRIPTION
**Pull request by mmattel**
The document how to get a space ID was faulty and outdated. This PR fixes this.

Backport to 5.0

### CODE REVIEW

**Review by kobergj — APPROVED** [review]

File: modules/ROOT/pages/maintenance/space-ids/space-ids.adoc [hunk 1]
```diff
@@ -1,84 +1,145 @@
...
-:description: There are rare situations like when using shell commands that require the ID of a space as parameter. This page shows how to get it.
+:description: There are rare situations like when using shell commands that require the space ID as parameter. This page shows how to get it.
 
 == Introduction
 
 {description} As an example of a use case, see the xref:deployment/services/s-list/storage-users.adoc#manage-trash-bin-items[Manage Trash-Bin Items] command set or the xref:{s-path}/search.adoc#manually-trigger-re-indexing-spaces[Manually Trigger Re-Indexing Spaces].
 
-* Space IDs can currently not be listed by the administrator via the WebUI. They can only be listed via the shell using an authenticated curl GET command accessing the Infinite Scale https://owncloud.dev/apis/[API].
-* End users can enable seeing the WebDAV Urls in their personal preferences of the Infinite Scale WebUI.
+. Via WebDav +
```
Comment (jnweiger) [inline modules/ROOT/pages/maintenance/space-ids/space-ids.adoc:9]:
```suggestion
. Via the WebUI +
For your personal space, open the file view and examine the URL in the browser.
For a manually created project space, open the file view, click on `Spaces`. Click on the desired space and examine the URL.

In either case, the browser URL contains a `fileId` parameter like e.g. this:

`...?fileId=c4dae5a1-7e34-47b2-947c-d9a3283f863f%24847593a2-e3e4-4e5f-ab44-d3d739a8a2ba!847593a2-e3e4-4e5f-ab44-d3d739a8a2ba&...`

The part after the exclamation mark `!` is the space ID.  ( `847593a2-e3e4-4e5f-ab44-d3d739a8a2ba` in this example.) 

. Via WebDav +
```

File: modules/ROOT/pages/maintenance/space-ids/space-ids.adoc [hunk 2]
```diff
@@ -1,84 +1,145 @@
...
-** Open an editor of choice where you can easily copy and paste to and add the following on top. The example lists `personal` users space ID's. Replace it with `project` for listing manually created spaces. Replace `<your host:9200>` with the URL:port of the Infinite Scale instance. Note to add a trailing blank line in the example as content is copied afterwards.
+** The assembled curl command from the editor will be executed here.
+
+* Prepare an editor. +
+The editor is used to assemble the command that is further copied to the console to get the final result. This is necessary because you need a valid bearer token with a limited lifetime you get from the browser.
+
+** Open an editor of choice. +
+Paste the following on top of the editor:
+
+*** Replace `<your host(:port)>` with the URL:port of your Infinite Scale instance. +
```
[the remark this answers is not in the document]
Comment (mmattel) [reply modules/ROOT/pages/maintenance/space-ids/space-ids.adoc:59]: You are right, we have to use square brackets.
The port is only necessary if one has defined it non-standard, but we have to note it

File: modules/ROOT/pages/maintenance/space-ids/space-ids.adoc [hunk 3]
```diff
@@ -1,84 +1,145 @@
...
 --
 
 * Open a browser
-** Login as administrator at `\https://<your host:9200>`, replace `<your host:9200>` with the URL:port of the Infinite Scale instance.
+
+** Login as administrator at `\https://<your host:9200>`
+*** Replace `<your host:9200>` with the URL:port of the Infinite Scale instance.
+*** Use the files view as starting point.
+
 ** Open the browsers menu:Developer Console[].
```
Comment (jnweiger) [inline modules/ROOT/pages/maintenance/space-ids/space-ids.adoc:87]:
```suggestion
** Open the browsers menu:Developer Console[] (in Firefox: More Tools -> Web Developer Tools[])
```

File: modules/ROOT/pages/maintenance/space-ids/space-ids.adoc (hunk 1 above)
Comment (mmattel) [reply modules/ROOT/pages/maintenance/space-ids/space-ids.adoc:9]: This is not correct. It is for all spaces and you must click on the space and enable the details view. Then you have to look on the `WebDAV Path/URL` to get the spaceID. I will improve the text

File: modules/ROOT/pages/maintenance/space-ids/space-ids.adoc [hunk 4]
```diff
@@ -1,84 +1,145 @@
...
 [source,bash]
 ----
-curl -XGET -vk 'https://<your host:9200>/graph/v1.0/drives?$filter=driveType+eq+personal' \
-  -H 'Authorization: Bearer ...
-| jq '.'
-
+curl -vk \
+  'https://<your host:9200>/graph/v1.0/drives?%24orderby=name%20asc&%24filter=driveType%20eq%20%27personal%27' \
+  -H 'accept: application/json' \
+  -H 'authorization: Bearer {token} \
```
Comment (jnweiger) [inline modules/ROOT/pages/maintenance/space-ids/space-ids.adoc:75]:
```suggestion
  -H 'authorization: Bearer {token}' \
```

File: modules/ROOT/pages/maintenance/space-ids/space-ids.adoc (hunk 1 above)
Comment (mmattel) [reply modules/ROOT/pages/maintenance/space-ids/space-ids.adoc:9]: I have also added a full section below explaining the process. Not necessary to do it here in the overview...

**Comment by jnweiger** [comment]
There is also https://github.com/owncloud/QA/blob/master/tools/hetzner-deploy/bin/ocis/find_spaces_fs.sh
to do the same from the command line.

**Comment by mmattel** [comment]
> The graph api also accepts basic auth

Basic auth is, as far I know, if you are not using the `auth-app`, disabled for security reasons 😱
There was an envvar ? I have to look...

**Comment by kobergj** [comment]
I agree with @mmattel. We should not use basic auth in examples as we advise not to use basic auth in production

**Review by jnweiger — APPROVED** [review]
the detour via bearer is horrible. 
Having this at one place is good.


approved.

**Pull request merged**